AirCasting/ABConnector/ConnectingABViewModel.swift

```diff
@@ -53,6 +64,16 @@ class AirbeamConnectionViewModelDefault: AirbeamConnectionViewModel, ObservableO
             }
         }
     }
+    
+    private func configureABforSync() {
+            let configurator = AirBeam3Configurator(userAuthenticationSession: self.userAuthenticationSession,
+                                                    peripheral: self.peripheral)
+            do {
+                try configurator.configureSDSync()
+            } catch {
+                Log.info("Couldn't configure AB for SD sync")
+            }
+    }
```

Comment:
<img width="965" alt="Screenshot 2021-11-24 at 09 10 08" src="https://user-images.githubusercontent.com/44276070/143208347-276cf759-b219-4c15-9c2e-a526300a968b.png">

Reply:
and ctrl+i please